src/main/groovy/org/shipkit/internal/gradle/git/GitBranchPlugin.java

```diff
@@ -0,0 +1,46 @@
+package org.shipkit.internal.gradle.git;
+
+import org.gradle.api.Action;
+import org.gradle.api.Plugin;
+import org.gradle.api.Project;
+import org.gradle.api.Task;
+import org.shipkit.gradle.git.IdentifyGitBranchTask;
+import org.shipkit.internal.gradle.util.TaskMaker;
+
+/**
+ * Adds tasks:
+ * <ul>
+ *     <li>identifyGitBranch - {@link IdentifyGitBranchTask}</li>
+ * </ul>
+ */
+public class GitBranchPlugin implements Plugin<Project> {
+
+    public static final String IDENTIFY_GIT_BRANCH = "identifyGitBranch";
```

Comment:
just a minor one: it looks like we have two `IDENTIFY_GIT_BRANCH` now (`GitPlugin#IDENTIFY_GIT_BRANCH` and this one).
Let's just have one of them.